### PR TITLE
Find Three Largest Number.

### PR DESCRIPTION
**Pull request by yashaswi90**

### CODE REVIEW

File: questions/src/main/java/com/ds/questions/FindThreeLargestNumber.java [hunk 1]
```diff
@@ -16,8 +16,19 @@ public static void main(String[] args) {
...
+            for (int i = 0; i < 3; i++) {
+                for (int j = i + 1; j <= inputData.length - 1; j++)
+                    if (inputData[j] > inputData[i]) {
+                        int temp = inputData[j];
+                        inputData[j] = inputData[i];
+                        inputData[i] = temp;
+                    }
+            }
+        }
+        return inputData;
```
Comment (aniketskulkarni) [inline questions/src/main/java/com/ds/questions/FindThreeLargestNumber.java]: Output should be only largest 3 numbers out of n numbers in `inputData`

File: questions/src/main/java/com/ds/questions/FindThreeLargestNumber.java [hunk 2]
```diff
@@ -16,8 +16,19 @@ public static void main(String[] args) {
     }
 
     private static int[] findThreeLargestNumber(int[] inputData) {
+        if (inputData == null || inputData.length == 0) {
+            return new int[]{0, 0};
```
Comment (aniketskulkarni) [inline questions/src/main/java/com/ds/questions/FindThreeLargestNumber.java]: Default output is [-1,-1], for whatever reason - not found, null, empty

**Review by aniketskulkarni — CHANGES_REQUESTED** [review]
Please work on the changes suggested

File: questions/src/main/java/com/ds/questions/FindThreeLargestNumber.java (hunk 2 above)
Comment (yashaswi90) [reply questions/src/main/java/com/ds/questions/FindThreeLargestNumber.java]: ok, thanks I will incorporate that comment.

File: questions/src/main/java/com/ds/questions/FindThreeLargestNumber.java (hunk 1 above)
Comment (yashaswi90) [reply questions/src/main/java/com/ds/questions/FindThreeLargestNumber.java]: Yes, In response I will return new array of three sizes only which has the three largest numbers from an array.

**Pull request merged**